build.yaml

```diff
@@ -142,7 +142,7 @@ steps:
      - from: /io/repo/git_version
        to: /git_version
      - from: /io/repo/address/test
-       to: /repo/address/test
+       to: /repo/address/
```

Comment:
Was this not used before?

Reply:
I don't know how the tests were working before. They had an extra layer of directories.